<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBookDialog</class>
 <widget class="QDialog" name="EditBookDialog">
  <property name="windowTitle">
   <string>Редактировать книгу</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelAuthor"><property name="text"><string>Автор:</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="authorLineEdit"/></item>
     <item row="1" column="0"><widget class="QLabel" name="labelTitle"><property name="text"><string>Название:</string></property></widget></item>
     <item row="1" column="1"><widget class="QLineEdit" name="titleLineEdit"/></item>
     <item row="2" column="0"><widget class="QLabel" name="labelYear"><property name="text"><string>Год:</string></property></widget></item>
     <item row="2" column="1"><widget class="QSpinBox" name="yearSpinBox"><property name="minimum"><number>0</number></property><property name="maximum"><number>2100</number></property><property name="value"><number>2000</number></property></widget></item>
     <item row="3" column="0"><widget class="QLabel" name="labelGenre"><property name="text"><string>Жанр:</string></property></widget></item>
     <item row="3" column="1"><widget class="QLineEdit" name="genreLineEdit"/></item>
     <item row="4" column="0"><widget class="QLabel" name="labelPublisher"><property name="text"><string>Издательство:</string></property></widget></item>
     <item row="4" column="1"><widget class="QLineEdit" name="publisherLineEdit"/></item>
     <item row="5" column="0"><widget class="QLabel" name="labelISBN"><property name="text"><string>ISBN:</string></property></widget></item>
     <item row="5" column="1"><widget class="QLineEdit" name="isbnLineEdit"/></item>
     <item row="6" column="0"><widget class="QLabel" name="labelPages"><property name="text"><string>Страниц:</string></property></widget></item>
     <item row="6" column="1"><widget class="QSpinBox" name="pagesSpinBox"><property name="minimum"><number>1</number></property><property name="maximum"><number>10000</number></property><property name="value"><number>300</number></property></widget></item>
    </layout>
   </item>
   <item><widget class="QDialogButtonBox" name="buttonBox"><property name="standardButtons"><set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>


